lib/publish.js

```diff
@@ -201,5 +193,15 @@ export default async (pluginConfig, context) => {
 
   const releaseUrl = urlJoin(gitlabUrl, projectPath, `/-/releases/${encodedGitTag}`);
 
+  if (publishToCatalog) {
+    try {
+      await execa("glab", ["repo", "publish", "catalog", gitTag], { cwd, timeout: 30 * 1000 });
```

Comment:
- This also introduces `execa` as a new dependency. What about just using `exec` from `node:child_process`? (`execa` is also not yet listed in `package.json` or here as import)
- Also `glab` respects `GITLAB_TOKEN` according to their [docs](https://docs.gitlab.com/editor_extensions/gitlab_cli/#authenticate-with-gitlab). But the plugin allows `GL_TOKEN` or `GITLAB_TOKEN`.  We should either add a check here and/or highlight this in the README.

Reply:
Thanks for the feedback @JonasSchubert!

Good point, I just saw `execa` used in other `semenatic-release` repos. I'll try with the standard lib instead.

Also good hint with the token, I think we need to pass on our variable to the child process.

I'll be out for the next two weeks, so don't expect any activity here :)

Reply:
> Good point, I just saw execa used in other semenatic-release repos. I'll try with the standard lib instead.

Just had another look. We're using [`execa` nearly everywhere](https://github.com/search?q=org%3Asemantic-release+execa%28&type=code), so let's go for consistency here.

Reply:
> Also glab respects GITLAB_TOKEN according to their [docs](https://docs.gitlab.com/editor_extensions/gitlab_cli/#authenticate-with-gitlab). But the plugin allows GL_TOKEN or GITLAB_TOKEN. We should either add a check here and/or highlight this in the README.

I have adapted the `execa` call to pass the token via the `GITLAB_TOKEN` variable.